source/dataman/DataMan.cpp

```diff
@@ -61,7 +61,10 @@ void DataMan::add_stream(json p_jmsg)
         man->init(p_jmsg);
         this->add_next(method, man);
     }
-    add_man_to_path("zfp", method);
+    if (p_jmsg["compress_method"] != nullptr)
```

Comment:
Rather than test for nullptr, just treat the pointer as a bool, i.e. `if(p_jmsg["compress_method"])`